plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/ContainerInfo.java

```diff
@@ -106,12 +107,12 @@ public void setNetworkSettings(NetworkSettings networkSettings) {
         this.networkSettings = networkSettings;
     }
 
-    public String getResolvConfPath() {
-        return resolvConfPath;
+    public String getResolveConfPath() {
```

Comment:
It is not a typo. We used the same wording Docker use itself. You can check it by inspecting docker container.